### PR TITLE
Implementation of the project structure and lint

### PR DESCRIPTION
**Pull request by ezequielsetzesglobant**
The project structure was implemented with clean architecture and lint was added to the project as well.

Clean Architecture structure:
Screen Shot 2022-04-19 at 10 06 16
Screen Shot 2022-04-19 at 10 06 49

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -44,7 +44,8 @@ dev_dependencies:
   # activated in the `analysis_options.yaml` file located at the root of your
   # package. See that file for information about deactivating specific lint
   # rules and activating additional ones.
-  flutter_lints: ^1.0.0
+  # flutter_lints: ^1.0.0
```
Comment (MarcosCGlobant) [inline pubspec.yaml]: you can remove the commented dependency

File: analysis_options.yaml [hunk 2]
```diff
@@ -7,7 +7,8 @@
 
 # The following line activates a set of recommended lints for Flutter apps,
 # packages, and plugins designed to encourage good coding practices.
-include: package:flutter_lints/flutter.yaml
+# include: package:flutter_lints/flutter.yaml
```
Comment (MarcosCGlobant) [inline analysis_options.yaml]: you can remove the commented reference

Comment (ezequielsetzesglobant) [reply analysis_options.yaml]: removed

File: pubspec.yaml (hunk 1 above)
Comment (ezequielsetzesglobant) [reply pubspec.yaml]: removed

**Review by LeonardoMolleker — APPROVED** [review]

**Review by Juliana-Dimeglio — APPROVED** [review]

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**